simtools/simtel/simtel_config_writer.py

```diff
@@ -104,7 +149,7 @@ def write_array_config_file(self, config_file_path, layout, telescope_model, sit
             file.write(self.TAB + "echo *****************************\n\n")
```

Comment:
I think we try to avoid Tabs as their length depend on local settings. In the current implementation, it is always exactly three spaces (I however do not know how sensitive sim_telarray is to spaces/tabs - this is Raul's or Orel's work).